website/content/docs/secrets/terraform.mdx

```diff
@@ -16,7 +16,7 @@ This page will show a quick start for this backend. For detailed documentation
 on every path, use `vault path-help` after mounting the backend.
 
 ~> **Terraform Enterprise Support:** this secret engine supports both Terraform
-Cloud ([app.terraform.io](https://app.terraform.io)) as well as on-premise
+Cloud ([app.terraform.io](https://app.terraform.io)) as well as on-premises
```

Comment:
We discussed this internally, and came up to the conclusion that it is better to use `on-prem` instead
```suggestion
Cloud ([app.terraform.io](https://app.terraform.io)) as well as on-prem
```